app/controllers/admin/config/about_controller.rb

```diff
@@ -5,5 +5,44 @@ def index
   end
 
   def update
+    settings_map = {}
+    if general_settings = params[:general_settings]
+      settings_map[:title] = general_settings[:name]
+      settings_map[:site_description] = general_settings[:summary]
+      settings_map[:about_banner_image] = general_settings[:about_banner_image]
+
+      settings_map[:extended_site_description] = general_settings[:extended_description]
+      if settings_map[:extended_site_description].present?
+        settings_map[:extended_site_description_cooked] = PrettyText.markdown(
```

Comment:
I think we should be using `PrettyText.cook` here?

Reply:
I looked at both methods and saw that `.cook` calls `.markdown` and then adds mentions and video support so I went with `markdown`. Should mention/videos be supported in the extended description?

Reply:
Good point, you are right that the simpler one is better :+1: